README.md

```diff
@@ -8,3 +8,18 @@ Originally created by the [HPC team of Ghent University](http://ugent.be/hpc).
 
 `vsc-mympirun` is made available under the GNU General Public License
 (GPL) version 2.
+
+
+# Workflow
```

Comment:
please split the lines in this markdown document in 80 characters. it makes it easier to comment the text